Cdms.Business.Tests/Cdms.Business.Tests.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="MartinCostello.Logging.XUnit" Version="0.4.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.8" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
+    <PackageReference Include="Moq" Version="4.20.72" />
```

Comment:
I think we are already using NSubstitute, so it will be best not to use different mocking frameworks within the tests